runtime/compiler/control/JITaaSCompilationThread.cpp

```diff
@@ -2298,6 +2300,22 @@ remoteCompile(
    std::string detailsStr = std::string((char*) &details, sizeof(TR::IlGeneratorMethodDetails));
    TR::CompilationInfo *compInfo = compInfoPT->getCompilationInfo();
    bool useAotCompilation = compInfoPT->getMethodBeingCompiled()->_useAotCompilation;
+   JITaaS::J9ClientStream *client;
+   if (enableJITaaSPerCompConn)
+      {
+      client = new (PERSISTENT_NEW) JITaaS::J9ClientStream(compInfo->getPersistentInfo());
```

Comment:
Can the connection establishment throw any exceptions?

Reply:
This line of code can throw "JITaaS::StreamFailure" and "std::alloc", will add appropriate handles